widgetastic_manageiq.py

```diff
@@ -878,3 +878,37 @@ def last_page(self):
     @property
     def items_amount(self):
         return self.paginator.page_info()[1]
+
+
+class ToolBarViewSelector(View):
+    """ represents toolbar's view selector control ("Grid View", "Tile View", etc)
+
+        .. code-block:: python
+        @View.nested
+        class view_selector(ToolBarViewSelector):  # NOQA
+
+        some_view.view_selector.select('Tile View')
+        some_view.view_selector.selected
+    """
+    ROOT = './/div[contains(@class, "toolbar-pf-view-selector")]'
+    BUTTONS = './/button'
+
+    @property
+    def _view_buttons(self):
+        br = self.browser
+        return [Button(self, title=br.get_attribute('title', btn)) for btn
+                in br.elements(self.BUTTONS)]
+
+    def select(self, title):
+        for button in self._view_buttons:
+            if button.title == title:
+                return button.click()
+        else:
+            raise ValueError('Incorrect button title passed')
```

Comment:
Consider something more like the following - I hesitate to call the title 'incorrect'.
'Given view title is not present: {}'.format(title)